### ISSUE TITLE
Armenian and Georgian unicode symbols missing

### ISSUE THREAD

**Issue by technic** [OpenPLi/enigma2#2377]
Hi,
The default font "Regular" is bound to nmsbd.ttf. This font is missing Armenian and Georgian language support, there is not corresponding Unicode symbols.
I think regular font shall support as much characters as possible, for example you can use DejaVu Sans

**Comment by WanWizard** [comment]
Isn't this a skin specific thing? If so, it's better to place the remark here: https://github.com/littlesat/skin-PLiHD /cc @littlesat

**Comment by technic** [comment]
I don't know where it belongs to, but the font files AFAIK come from this repo while the definition is in the other repository.

**Comment by WanWizard** [comment]
Ok. Let's wait for @littlesat to respond.

**Comment by WanWizard** [comment]
@littlesat ?

**Comment by WanWizard** [comment]
@littlesat ???

**Comment by persianpros** [comment]
Yes https://github.com/OpenPLi/enigma2/blob/develop/data/fonts/DejaVuSans.ttf almost supports everything but I think we need to have an option for it:

- Font changer

I've done it before via shell not python and it's going to be in my upcoming plugin + its android controller so even with your android device you could change the font, even if your skin goes to crash the smart crash detector will auto restore the skin backup after exactly 60 seconds.

This is complex and I'm not sure how we can manage it via python but I think the second idea would be better.

- Font overwrite

We need to add an option in settings which could select ttf fonts that exist in https://github.com/OpenPLi/enigma2/tree/develop/data/fonts then add a python function to read its value and if it's not empty overwrites whatever is in skin.xml regarding the ttf fonts.

@IanSav 

What do you think about the second option?

**Comment by technic** [comment]
@persianpros 
The skin is often designed to be used with the specific font, because size of widgets are related to size of font characters. The regular font must be updated to support all languages (as many as possible), but overriding random skin fonts by some plugin is bad. There is a solution like metrix-hd skin where you can tweak almost everything. But for the core image, keep it simple.

**Comment by IanSav** [comment]
Yes, skins can have a major effect on the fonts selected and used.

In this case I would tend to agree with @technic that the Regular font could be updated to provide a more language friendly alternative.  I also use DejaVu as a font for all my multilingual screens.
